contracts/ainft/AINFTBaseV2.sol

```diff
@@ -0,0 +1,224 @@
+// SPDX-License-Identifier: UNLICENSED
+// AINFT Contracts v2.0.0
+pragma solidity ^0.8.9;
+
+import {Strings} from '@openzeppelin/contracts/utils/Strings.sol';
+import '@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol';
+import '@openzeppelin/contracts/token/ERC721/extensions/ERC721Burnable.sol';
+import '@openzeppelin/contracts/access/Ownable.sol';
+import '@openzeppelin/contracts/access/AccessControl.sol';
+import 'operator-filter-registry/src/DefaultOperatorFilterer.sol';
+import '../interfaces/IAINFTBaseV1.sol';
+
+/**
+ * @title AINFTBaseV2
+ *
+ * @dev Implementation of the [ERC721](https://eips.ethereum.org/EIPS/eip-721)
+ * Non-Fungible Token Standard, overriding transfer and approval methods.
+ * Only addresses filtered by OpenSea's [Operator Filter Registry](https://github.com/ProjectOpenSea/operator-filter-registry) are allowed.
+ *
+ * Token IDs are minted in sequential order (e.g. 1, 2, 3, ...)
+ *
+ * Assumptions:
+ *
```

Comment:
empty line that can be removed?

Reply:
Yes, I removed empty line and unnecessary information 🙂